tests/ScopeTest.php

```diff
@@ -275,45 +275,45 @@ public function testScope()
     {
         $user = clone $this->user;
 
-        $condition1 = new Condition('name', 'John');
+        $condition1 = ['name', 'John'];
```

Comment:
this is why I do not think we should support array input, but this is short test, ok now